lib/ReactViews/FeatureInfo/FeatureInfoSection.jsx

```diff
@@ -86,6 +87,18 @@ export const FeatureInfoSection = observer(
     getTemplateData() {
       const propertyData = this.getPropertyValues();
       if (defined(propertyData)) {
+        // Alises is a map from `key` (which exists in propertyData.properties) to some `aliasKey` which needs to resolve to `key`
+        // and Yes, this is awful, but not that much worse than what was done in V7
+        let aliases = [];
+        if (TableMixin.isMixedInto(this.props.catalogItem)) {
+          aliases = this.props.catalogItem.columns
+            .filter(col => col.name && col.title && col.name !== col.title)
+            .map(col => [col.name, col.title]);
+        }
+        aliases.forEach(aliasMap => {
+          propertyData[aliasMap[0]] = propertyData[aliasMap[1]];
+        });
+
```

Comment:
Should `aliasMap[1]` keys be removed from `propertyData`? And is it possible or likely that an `aliasMap[0]` could also be an `aliasMap[1]` later in `aliases` and hence the key will be overwritten in `propertyData`?

Reply:
I think we need both in `propertyData` - as some feature templates use column name and others use column title (even if both are defined).
Overwriting aliases is definitely possible, but it seems unlikely someone would name a `column.title` after a different `column.name`. I have added this so at least we aren't overriding properties which are defined:

```js
propertyData[aliasMap[0]] = propertyData[aliasMap[0]] ?? propertyData[aliasMap[1]];
```

I have also shallow cloned `propertyData` so we aren't messing with `feature.propertes|currentProperties|...`